src/pom.xml

```diff
@@ -135,8 +135,9 @@
         <sparkbench.mr.version>2</sparkbench.mr.version>
       </properties>
       <activation>
+        <activeByDefault>true</activeByDefault>
```

Comment:
If people want to build with MR1, they should specify with `-P MR1'. I think MR2 is more popular now, so I make it as default. Does any people still work with MR1 as you know?